scripts/cut_release_or_hotfix_branch.py

```diff
@@ -166,29 +216,56 @@ def _execute_branch_cut():
     common.open_new_tab_in_browser_if_possible(
         'https://github.com/oppia/oppia#oppia---')
     while True:
+        if not HOTFIX_NUMBER:
+            branch_to_check = 'develop'
+        elif HOTFIX_NUMBER == 1:
+            branch_to_check = 'release-%s' % TARGET_VERSION
+        else:
+            branch_to_check = 'release-%s-hotfix-%s' % (
+                TARGET_VERSION, HOTFIX_NUMBER - 1)
         python_utils.PRINT(
-            'Please confirm: are Travis checks passing on develop? (y/n) ')
+            'Please confirm: are Travis checks passing on %s? (y/n) ' % (
+                branch_to_check))
         answer = python_utils.INPUT().lower()
         if answer in release_constants.AFFIRMATIVE_CONFIRMATIONS:
             break
         elif answer:
             python_utils.PRINT(
-                'Tests should pass on develop before this script is run. '
-                'Exiting.')
+                'Tests should pass on %s before this script is run. '
+                'Exiting.' % branch_to_check)
             sys.exit()
 
-    # Cut a new release branch.
-    python_utils.PRINT('Cutting a new release branch: %s' % NEW_BRANCH_NAME)
-    subprocess.call(['git', 'checkout', '-b', NEW_BRANCH_NAME])
+    if HOTFIX_NUMBER:
+        new_branch_type = 'hotfix'
+    else:
+        new_branch_type = 'release'
```

Comment:
Maybe put `'hotfix'` and `'release'` into constants.

Reply:
Done.